src/6model/reprs.c

```diff
@@ -196,7 +196,7 @@ void MVM_repr_initialize_registry(MVMThreadContext *tc) {
 
     /* Add all core representations. */
     register_core_repr(String);
-    register_core_repr(Array);
+    register_repr(tc, VMArray_initialize(tc), NULL);
     register_core_repr(Hash);
```

Comment:
I'm not familiar with this codebase.... but I notice all the other changes are merely renamings, but this particular change seems to replace code line with something entirely different. Is that meant to be that way?